djangocms_references/views.py

```diff
@@ -3,7 +3,7 @@
 from django.utils.translation import ugettext_lazy as _
 from django.views.generic.base import TemplateView
 
-from .helpers import get_all_reference_objects
+from .helpers import get_extra_columns, get_all_reference_objects
 
 
 class ReferencesView(TemplateView):
```

Comment:
I'm not confident that this view is properly tested. Had a quick glance at the tests there previously and they look a bit weird - instantiating the view class rather than doing `self.client.get` calls, which gives me little confidence because if, for example, something changed in the dispatch method that made the view never go into `get_context_data` the fact those tests would be passing is meaningless.
For this PR I definitely think there should be a test which does an http call, then checks the context exactly by checking `response.context_data`. From what I can see current tests only check that variables appear in the context, but if they were all set to None the test would still pass. This is not enough to give us confidence things are working.
You probably want two tests. One which checks the whole context data (including that all the extra column data is there) with a versioned model. And then a second one which checks this for an unversioned model (I think it's enough if it checks solely the queryset and extra column headers as those are the only parts of the context effected by versioning, yes?).